test/e2e/swaps/shared.js

```diff
@@ -25,10 +25,8 @@ const loadSwaps = async (driver) => {
 };
 
 const buildQuote = async (driver, options) => {
-  await driver.clickElement(
-    '[class*="dropdown-search-list"] + div[class*="MuiFormControl-root MuiTextField-root"]',
-  );
   await driver.fill('input[placeholder*="0"]', options.amount);
+  await driver.delay(1000); // Need an extra delay after typing an amount.
```

Comment:
Instead of this arbitrary second, could you use one of the variables in `test/e2e/helpers.js`?  Maybe `largeDelayMs`?

Reply:
Maybe this can be done in a follow-up PR, since this is blocking all other PRs and the release that we're preparing, and any change would require another ~30 minute wait.